src/mumble/MenuLabel.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_MENULABEL_H_
+#define MUMBLE_MUMBLE_MENULABEL_H_
+
+#include <QString>
+#include <QtWidgets/QWidgetAction>
```

Comment:
```suggestion
#include <QWidgetAction>
```
Including Qt stuff from "subdirectories" is discouraged